clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -140,6 +146,9 @@ def err_drv_incompatible_unwindlib : Error<
   "--rtlib=libgcc requires --unwindlib=libgcc">;
 def err_drv_incompatible_options : Error<
   "The combination of '%0' and '%1' is incompatible">;
+def err_drv_incompatible_fp_accuracy_options : Error<
+  "fp-accuracy requirements cannot be guaranteed when math-errno is enabled. "
+  "Use -fno-math-errno to enable fp-accuracy control.">;
```

Comment:
This doesn't match our usual diagnostic requirements. 1) reword fp-accuracy as above, 2) math-errno should be `'-fmath-errno'` 3) change to: `... is enabled; use '-fno-math-errno' to ... ` and drop the trailing full stop.